webpack.mix.js

```diff
@@ -29,6 +31,16 @@ if (mix.inProduction()) {
   min = '.min';
 }
 
+const paymentSandbox = function () {
+  if (process.env.PAYMENT_SANDBOX == 0
```

Comment:
What a fancy way to say `paymentSandbox = x != 0 && x !== 'false' && !x`. Also is that `x == 0` what does that even mean.